gulpfile.js

```diff
@@ -74,6 +76,8 @@ function bumpTask() {
             targetVersion = (res.type !== 'custom') ? semver.inc(currentVersion, res.type) : res['custom-version'];
             responses = res;
 
+            bumpPackageLock(targetVersion);
```

Comment:
There is a simpler approach to this. Look at the readme for gulp-bump: https://github.com/stevelacy/gulp-bump#bumping-version-and-outputting-different-files